### ISSUE TITLE
Only generate noise after the first and second stage training in AE-Adapt-V?

### ISSUE THREAD

**Issue by k15201363625** [dc-ai-projects/DC-VideoGen#7]
Thanks for your great work. And we try to reproduce the paper and struggle in generating normal videos after two-stage AE-Adapt-V training.

In the first stage, we only training patch embedding to align the latent space. In the second stage, we training the patch embedding and output head using Diffusion Loss. 

As the figure 6(b) and figure 7(a) in the main paper, the model is expected to generate normal videos which is not noise with some artifacts. But We only can generate noise after 4000 steps training. 

Details: we use LTX-vae and Wan2.1 1.3B video diffusion model. We use 100k video datas with 480x832 resolution and training with 81 frames.

The first stage training curve:

<img width="612" height="372" alt="Image" src="https://github.com/user-attachments/assets/f3deb933-81a4-4b9e-ba87-fb2631be3e44" />

As the blue curve and yellow curve, the mse loss has been lower than 0.01 with batch_size=64 and 1000 training steps.

The second stage training curve (use checkpoints in yellow or blue curve get similar training curve in the second stage.):

<img width="390" height="350" alt="Image" src="https://github.com/user-attachments/assets/269b7b8d-b0c1-49b8-b3d5-25689a0dd1e2" />

And the training loss in the second stage decrease from 1.x to 0.1. 

In the two-stage training, we fix the dit model following setting in the paper. After the two-stage adaptive training, we load trained weights of patch embedding and output head, and generate videos:

https://github.com/user-attachments/assets/3f4133ad-9b24-4480-8681-43e3a64b85b1

this is pure noise, but when we use original wan2.1 vae, patch embedding and output head, we can generate normal video:

https://github.com/user-attachments/assets/44c4f357-2a55-4f45-8b01-a18bd311107a

According your experience, what is the possible problem? And are there more details for your introduced new patch embedding and output head, they are same with the original structure(one layer conv3d and one layer mlp)? How to judge training is finished in some stage, loss curve? 

Expect your response, Thanks.

**Comment by kingofprank** [comment]
@k15201363625 I encountered similar issues when trying to reproduce it. I trained using 640p-45f data. In the first stage, the MSE loss was approximately 0.17, and in the second stage, the diffusion loss was between 0.5 and 0.7. In the third stage (using full SFT), the diffusion loss continued to decrease, but the final convergence result showed a noticeable artifact. Have you solved this problem? Do you have any key techniques you could share?

**Comment by k15201363625** [comment]
> [@k15201363625](https://github.com/k15201363625) I encountered similar issues when trying to reproduce it. I trained using 640p-45f data. In the first stage, the MSE loss was approximately 0.17, and in the second stage, the diffusion loss was between 0.5 and 0.7. In the third stage (using full SFT), the diffusion loss continued to decrease, but the final convergence result showed a noticeable artifact. Have you solved this problem? Do you have any key techniques you could share?

hi, I try experiments with different setting, including using more complex patch embedding and output head architectures, different training hyperparameters, and training data from various sources(openvid and wan2.1 generated videos). However, the best-performing configuration so far still produces videos with noticeable artifacts.

Specifically, after the second training stage(batch_size=64, lr=1e-4, 10k steps), the generated video **match** the result in paper:


https://github.com/user-attachments/assets/8f0f02d8-8cf2-4f84-bc09-990cb633e22a

https://github.com/user-attachments/assets/9d564b1b-eaf9-44ae-93e4-0a7f8fd82ba7

But, in the third stage, neither full-parameter fine-tuning nor DiT LoRA (with lora_rank=256 and alpha=128) is able to further improve the generation quality. In fact, the generated videos often become worse than those from the second stage, after 20k or 40k steps training. This does **not match** the result in paper:

https://github.com/user-attachments/assets/fa076aea-1866-46b2-9448-d2ddeaa44dd1

https://github.com/user-attachments/assets/526de1eb-9968-4ee0-b871-296d8f714a4e


Although the training loss continues to decrease and appears to have converged, following the training setting specified in the appendix of the paper still fails to produce normal-looking videos. There may be missing details or implementation issues. Expect the guidance from authors, thanks.

**Comment by k15201363625** [comment]
Since currently the project cannot be reproduced and there has been no feedback from authors, issue is closed.
